spoc/io.py

```diff
@@ -4,7 +4,10 @@
 from typing import Dict, Union
 import pandas as pd
 import dask.dataframe as dd
-from .models import FragmentSchema, AnnotatedFragmentSchema, HigherOrderContactSchema
+from .contacts import Contacts
+from .pixels import Pixels
+from .dataframe_models import FragmentSchema, ContactSchema
+from .fragments import Fragments
 
 
 class FileManager:
```

Comment:
pickling label library feels unsafe &mdash; protocols and versions might change with python versions. Also doesn't allow people to generate files with other languages (e.g., R). Maybe some kind of json, yaml or parquet?

Reply:
Yeah, good point! Created a ticket to fix this: https://trello.com/c/KtXGzS6n/81-make-persistence-of-labellibrary-use-a-standard-format